outputs.tf

```diff
@@ -1,3 +1,8 @@
+output "account_id" {

```

Comment:
Why do we need that?

Reply:
To be able to use the output in calling modules, e.g. we use(d) it to generate a command output for getting the _kubeconfig_ that required the account id.

Reply:
have removed it again